src/picterra/client.py

```diff
@@ -386,10 +388,36 @@ def download_result_to_file(self, operation_id, filename):
             r.raise_for_status()
             with open(filename, 'wb') as f:
                 logger.debug('Trying to save result to file %s..' % filename)
-                for chunk in r.iter_content(chunk_size=8192):
+                for chunk in r.iter_content(chunk_size=CHUNK_SIZE_BYTES):
                     if chunk:  # filter out keep-alive new chunks
                         f.write(chunk)
 
+    def download_url_to_file(self, operation_id, filename):
```

Comment:
Would rename this to `download_operation_results_to_file`, because I would expect `download_url_to_file` to take an url as input.

Reply:
Checked